root/frontend/src/routes/Home.tsx

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
 
+import { ThemeProvider } from 'react-jss';
 import Calendar from '../components/Calendar/Calendar';
+import PreviewCard from '../components/PreviewCard/PreviewCard';
 
 function Home() {
+  //  placeholder until we populate database with exercises + theme colors
```

Comment:
I don't believe the cards need to know anything regarding the database info, since it's a display component and not really one that handles logic.

Reply:
Yeah, I meant that I was placing the component here to see what it would look like. This would be replaced with a different component that handles the mapping of the preview cards later.